.gitlab-ci.yml

```diff
@@ -238,6 +254,8 @@ init_workspace:
   tags:
     - mender-qa-slave
   before_script:
+    # Default value, will later be overwritten if successful
+    - echo "failure" > ${CI_PROJECT_DIR}/JOB_RESULT.txt
```

Comment:
For the tests jobs this cannot be done this early: we are wiping out CI_PROJECT_DIR and restoring the workspace tarball at the beginning of the `script` section. There is where it should be done.

Reply:
(right now this only affects build and acceptance testing, but after PR #290 it will only affect integration and backend integration tests)

Reply:
Nice catch, I will fix that.

Reply:
In order to avoid a fragile approach, I decided to replace all instances with `${HOME}/JOB_RESULT.txt` instead. I think only one job will run inside one Docker container anyway, so it should be safe, and should be able to survive wiping out the `WORKSPACE`, wherever we do it.